.github/CONTRIBUTING.md

```diff
@@ -3,15 +3,20 @@
 
 If you are unfamiliar with the workflow of contributing to github, you can refer to this [this article](https://github.com/firstcontributions/first-contributions/blob/master/README.md)
 
-## Fork & clone this repository
+## External contributions 
 
-The development should happen in a personal fork, cloned on the local machine.
+If you are not part of the team and want to contribute to this repository, you must fork & clone this repository
 
-## Use development branch
+The development should happen in a personal fork, cloned on the local machine. Make sure you use signed commits before opening
+a pull request.
 
-External contributions should happen against the `development` branch.
+For external contributors, the PRs should be targeted towards the `master` branch. A team responsible will instruct 
+the PR owner to re-target it against another branch, in accordance to internal branches management.
 
-Other branches may be used by the team members in accordance to internal decisions.
+tl;dr:
+- fork `elrond-go` and use `master` branch
+- use signed commits
```

Comment:
Perhaps add a link to:

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

Reply:
added